setup.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
```

Comment:
I'd suggest making setup.py executable, and adding a `#!/usr/bin/python` at the top.